lib/diagnostic.rb

```diff
@@ -105,15 +124,21 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+
+  def roar
+    puts "ROAR!"
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# Mixins are used when things "behave as" rather than "is a" part of.
```

Comment:
Classes can include any number of modules, but classes can only inherit from one other classes.